parsers/test/test_BG.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta, timezone
+
+import pytest
+import requests_mock
+from freezegun import freeze_time
+
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
```

Comment:
This is a symlink which might break on windows (we have had some trouble with that in the past) and it should be imported from just `parsers.lib.exceptions` instead. (Sadly we can't remove the symlink due to internal uses.)